esphome/components/nextion/display.py

```diff
@@ -133,6 +137,9 @@ async def to_code(config):
     var = cg.new_Pvariable(config[CONF_ID])
     await uart.register_uart_device(var, config)
 
+    if (max_queue_age := config.get(CONF_MAX_QUEUE_AGE, None)) is not None:
+        cg.add(var.set_max_queue_age(max_queue_age))
```

Comment:
```suggestion
    cg.add(var.set_max_queue_age(config[CONF_MAX_QUEUE_AGE]))
```

since they have defaults, they could be set unconditionally and will never be `None`